tools/pd-ctl/pdctl/command/cluster_command.go

```diff
@@ -47,19 +43,21 @@ func NewClusterStatusCommand() *cobra.Command {
 }
 
 func showClusterCommandFunc(cmd *cobra.Command, args []string) {
-	r, err := doRequest(cmd, clusterPrefix, http.MethodGet, http.Header{})
+	cli := pd.NewClient(pdControlCallerID, getEndpoints(cmd))
```

Comment:
Since the internal of pd-ctl will definitely share the same PD HTTP Client instance in the end, can we initialize and pass it to each subcommand from a more central position? For example, as a singleton global variable.

Reply:
pd-ctl supports passing the parameter pdAddrs, which means it's a way to set `NewClient` dynamically.

It might be possible to support global functions reading addrs instead of global variables.